src/Modal/Modal.js

```diff
@@ -138,8 +140,8 @@ Modal.defaultProps = {
   showClose: true,
   closeOnBackdropClick: true,
   closeOnEscape: true,
-  minWidth: 250,
-  maxWidth: 768,
+  minWidth: 350,
+  maxWidth: 350,
```

Comment:
What prompted this `maxWidth` change to `350`? The camera modal in the `web-app` takes up a lot more space than `350` and I think we want to keep it that way. 😄

Reply:
The camera modal is the only exception in this case. Which is why I decided to change the default `maxWidth` to 350 which is what Jay specified [here](https://app.asana.com/0/1129589982637900/1132970972936105).

All modals can override this in their props.